### ISSUE TITLE
i18n issues in SliderBox

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/resistance-in-a-wire#59]
(1) Hardcoded strings should be translatable:

`51 this.addChild( new Text( 'ρ', {`
`82 this.addChild( new Text( 'L', {` 
`113 this.addChild( new Text( 'A', {`

(2) Ordering should be determined by using a pattern (e.g. `{0} {1}`), not via string concat:

`73 this.addChild( new Text( 'Ω' + cmString.`
`135 this.addChild( new Text( cmString + '²', {`

(3) Superscript should be done via SubSupText, not a literal '2' superscript:

`135 this.addChild( new Text( cmString + '²', {`

**Comment by pixelzoom** [comment]
https://github.com/phetsims/resistance-in-a-wire/issues/43 is related to this, and mentions other i18n issues.

**Comment by jbphet** [comment]
I did one thing in the process of addressing this issue that is perhaps a bit unconventional.  I made the symbols translatable, but if a translator attempts to use more than one character for a symbol, the string will be truncated to the the first character.  This was a lot easier than changing the FormulaView class to handle arbitrary strings, with didn't seem worth the effort.  This was discussed with @ariel-phet and approved.
